samples/index_tuning_sample/README.md

```diff
@@ -0,0 +1,312 @@
+
+This tutorial is hosted on [LangChain AlloyDB integration library](https://github.com/googleapis/langchain-google-alloydb-pg-python/tree/main/samples/index_tuning_guide) in Markdown format, accompanied by Python code sample files.
+
+## Before You Begin
+
+1. Make sure you have a Google Cloud project and billing is enabled.
+1. Install the gcloud CLI.
+
+    Set gcloud project:
+
+    ```bash
+    gcloud config set project $PROJECT_ID
+    ```
+
+1. Enable APIs:
+
+    ```bash
+    gcloud services enable alloydb.googleapis.com
+    gcloud services enable compute.googleapis.com
+    gcloud services enable cloudresourcemanager.googleapis.com
+    ```
+
+1. If you haven't already, create an AlloyDB database cluster and instance following this [guide](https://www.google.com/url?q=https://cloud.google.com/alloydb/docs/quickstart/create-and-connect&sa=D&source=docs&ust=1719329998287572&usg=AOvVaw0wZYIMI4to2KljZjjDnsdv).
+
+## Step 1: Clone sample code
+
+Run `git clone` command in your local directory to pull the sample code:
+
+```bash
+git clone <https://github.com/googleapis/langchain-google-alloydb-pg-python>
+```
+
+## Step 2: Move into your local sample directory
+
+Move into the sample directory:
+
+```bash
+cd samples/index_tuning_sample
+```
+
+## Step 3:  Install Dependencies
+
+In your local sample code directory, run this command to install required dependencies:
+
+```bash
+pip3 install -r requirements.txt
+```
+
+## Step 4: Fill In Database Info
+
+Enter your database information and credentials in `create_vector_embeddings.py`:
+
+```python
+AlloyDB info
+PROJECT_ID = ""
+REGION = ""  # @param {type:"string"}
+CLUSTER_NAME = ""  # @param {type:"string"}
+INSTANCE_NAME = ""  # @param {type:"string"}
+DATABASE_NAME = ""  # @param {type:"string"}
+USER = ""  # @param {type:"string"}
+PASSWORD = ""  # @param {type:"string"}
+```
+
+## Step 5: Generate Embeddings
+
+If you already have vector embeddings ready in your database, you can skip this step.
+Otherwise, create embeddings from the sample dataset `wine_reviews_dataset.csv` by running this command in your sample code directory:
+
+```bash
+python3 create_vector_embeddings.py
+```
+
+Now your database is populated with 100k vector embeddings.
+
+## Step 6: Create a Database Vector Store
+
+we create a database engine from the information you input previously, and create a vector store from that engine:
+
+```python
+from langchain_google_alloydb_pg import (
+    AlloyDBEngine,
+    AlloyDBVectorStore,
+)
+
+from langchain_google_alloydb_pg.indexes import DistanceStrategy
+
+DISTANCE_STRATEGY = DistanceStrategy.COSINE_DISTANCE
```

Comment:
I don't want us to set distance strategy. Let's just add a note at the end that it can also be configured